windows/wintools_complete_install_verify/wintools_complete_install_verify.yml

```diff
@@ -29,12 +29,14 @@
         - include_tasks: ../../common/skip_test_case.yml
           vars:
             skip_msg: "Skip test case due to under development VMware tools can not test on VM with VBS enabled."
+            skip_reason: "Not Supported"
```

Comment:
I think this can be "Not Applicable" since in the current config situation the test case can not run.